Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:stretch
+MAINTAINER reswitched
+
+RUN echo "deb http://apt.llvm.org/stretch/ llvm-toolchain-stretch main \n\
+deb-src http://apt.llvm.org/stretch/ llvm-toolchain-stretch main \n\
+# 4.0 \n\
+deb http://apt.llvm.org/stretch/ llvm-toolchain-stretch-4.0 main \n\
+deb-src http://apt.llvm.org/stretch/ llvm-toolchain-stretch-4.0 main \n\
+# 5.0 \n\
+deb http://apt.llvm.org/stretch/ llvm-toolchain-stretch-5.0 main \n\
+deb-src http://apt.llvm.org/stretch/ llvm-toolchain-stretch-5.0 main" >> /etc/apt/sources.list
+
+RUN apt-get update
+RUN apt-get install -y git build-essential wget squashfs-tools python python-pip
+RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
+
+RUN apt-get update
```

Comment:
Is there a reason we run update again here? Can it be combined with the above?

Reply:
Best practice for running APT in docker is to run `apt-get update` and `apt-get install ...` all in a one `RUN` line so it can be cached in a single layer :)

Reply:
My only reasoning was that I needed to install `wget` first to get the gpg keys for the llvm repos, and then had to update again to actually get llvm.

Maybe i can put their public key right in the Dockerfile?